docs/services/vision/detection.md

```diff
@@ -0,0 +1,306 @@
+
+{{% alert title="Note" color="note" %}}
+
+**hue_tolerance_pct**, **saturation_cutoff_pct**, and **value_cutoff_pct** refer to hue, saturation, and value (brightness) in the HSV Color Model, but do not set color values in Viam.
+
+**hue_tolerance_pct** specifies the exactness of the color match to **detect_color**.
+
+The optional **saturation_cutoff_pct** and **value_cutoff_pct** attributes specify cutoff thresholds levels for saturation and brightness, rather than specifying color saturation and brightness as they do in the standard HSV Color Model.
+
+{{% /alert %}}
+
+### TFLite detector
+
+A machine-learning based detector that draws bounding boxes according to the specified tensorflow-lite model file available on the robot’s hard drive.
+
+These are the available parameters in the detector's configuration. For an example see [Configuration](#configuration).
+
+``` json {class="line-numbers linkable-line-numbers"}
+{
+    "register_models": [
+        {
+            "name": "<detector_name>",
+            "type": "tflite_classifier",
+            "parameters": {
+            "model_path" : "/path/to/model.tflite",
+            "label_path": "/path/to/labels.txt",
+            "num_threads": <number>
+            }
+        }
+    ]
+}
+```
+
+| Parameter | Description |
+| --------- | ----------- |
+| `model_path`| Required. The path to the `.tflite model` file, as a `string`. |
+| `num_threads`| An integer that defines how many CPU threads to use to run inference. The default value is 1. |
+| `label_path`| The path to a `.txt` file that holds class labels for your TFLite model, as a `string`. The SDK expects this text file to contain an ordered listing of the class labels. Without this file, classes will read as "1", "2", and so on. |
+
+#### TFLite Model Limitations
+
+We strongly recommend that you package your `.tflite` model with metadata in [the standard form](https://github.com/tensorflow/tflite-support/blob/560bc055c2f11772f803916cb9ca23236a80bf9d/tensorflow_lite_support/metadata/metadata_schema.fbs).
+
+In the absence of metadata, your TFLite model must satisfy the following requirements:
+
+- A single input tensor representing the image of type UInt8 (expecting values from 0 to 255) or Float 32 (values from -1 to 1).
+- At least 3 output tensors (the rest won’t be read) containing the bounding boxes, class labels, and confidence scores (in that order).
+- Bounding box output tensor must be ordered [x x y y], where x is an x-boundary (xmin or xmax) of the bounding box and the same is true for y.
+  Each value should be between 0 and 1, designating the percentage of the image at which the boundary can be found.
+
+These requirements are satisfied by a few publicly available model architectures including EfficientDet, MobileNet, and SSD MobileNet V1.
+You can use one of these architectures or build your own.
+
+## Configuration
+
+### Add the service and detector
+
```

Comment:
It seems like detection.md gets this section about configuration, but classification does not. It would be useful if classification also had a section like it - or if it were put at the general level.

Reply:
Yes - I wanted to run this by you before adding creating all the pages - just in case it wasn't a good idea for some reason